src/app.js

```
@@ -1 +1,62 @@
+/* eslint-disable no-console */
 // write code here
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFiles = () => {
+  const params = process.argv.slice(2);
+
+  if (params.length < 2) {
+    console.error('There are missing arguments');
+
+    return;
+  }
+
+  const source = params[0];
+  const destination = params[1];
+
+  if (!fs.existsSync(source)) {
+    console.error('Source not exist');
+
+    return;
+  }
+
+  if (!fs.statSync(source).isFile()) {
+    console.error('Source is not a file');
+
+    return;
+  }
+
+  let finalDestination = destination;
+
+  if (fs.existsSync(destination) && fs.statSync(destination).isDirectory()) {
+    const fileName = path.basename(source);
+
+    finalDestination = path.join(destination, fileName);
+  } else if (destination.endsWith('/')) {
+    console.error('Destination directory does not exist');
+
+    return;
+  } else {
+    const destDir = path.dirname(destination);
+
+    if (!fs.existsSync(destDir)) {
+      console.error('Destination directory does not exist');
+
+      return;
+    }
+  }
+
+  if (source === finalDestination) {
+    return;
```

According to the previous review and task description, when source and destination are the same, you should log an error to the console instead of silently returning.